database/seeders/VoucherSeeder.php

```diff
@@ -37,6 +37,14 @@ public function run()
                 'start_date' => date('Y-m-d', strtotime('09/28/2023')),
                 'end_date' => date('Y-m-d', strtotime('01/01/2025')),
             ],
+            [
+                'voucher_code' => 'KM1111',
+                'type' => 'direct',
+                'amount' => 1000,
+                'quantity' => 10,
+                'start_date' => date('Y-m-d', strtotime('11/11/2023')),
+                'end_date' => date('Y-m-d', strtotime('11/11/2023')),
+            ],
```

Comment:
I think we need more voucher for testing(checkout feature is the highest important thing in our system)